patches/common/chromium/gritsettings_resource_ids.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/tools/gritsettings/resource_ids b/tools/gritsettings/resource_ids
+index 3539e7ce5ab4..9450ee117e83 100644
+--- a/tools/gritsettings/resource_ids
++++ b/tools/gritsettings/resource_ids
+@@ -400,6 +400,9 @@
+   "devtools_resources.grd": {
+     "includes": [28750],
+   },
++  "electron/electron_resources.grd": {
++    "includes": [31750],
```

Comment:
Other than the fact that the number should be greater than 31000, no special reason for this number, right?

Reply:
Yup that is the only reason.